ushahidi/platform-client-mzima
# Ush 1589 - View Multimedia Content (Mobile)

**Issues:**

Users should be able to view posts with new multimedia content: images, audio and documents on mobile.

Update: PR ignores editing new multimedia fields on Mobile.


**Solution:**

This PR adds handling for multimedia fields in the view post templates on mobile.

**Testing:**

1. Create a post with multimedia content (on web).
2. Open the deployment with the mobile app.
3. View the post and note all the multimedia content is displayed correctly.
4. Edit the post and note that the multimedia content does not appear and post can be saved.
